code/modules/jobs/job_types/chaplain/chaplain.dm

```diff
@@ -78,15 +78,17 @@
 				else
 					holy_bible.deity_name = pick("Gay Space Jesus", "Gandalf", "Dumbledore")
 			human_spawned.adjustOrganLoss(ORGAN_SLOT_BRAIN, 100) // starts off brain damaged as fuck
-		if("lol", "wtf", "poo", "badmin", "shitmin", "deadmin", "meme", "memes")
-			new_bible = pick("Woody's Got Wood: The Aftermath", "Sweet Bro and Hella Jeff: Expanded Edition","F.A.T.A.L. Rulebook")
+		if("lol", "wtf", "poo", "badmin", "shitmin", "deadmin", "meme", "memes", "skibidi")
+			new_bible = pick("Woody's Got Wood: The Aftermath", "Sweet Bro and Hella Jeff: Expanded Edition","F.A.T.A.L. Rulebook", "Toilet Humor")
 			switch(new_bible)
 				if("Woody's Got Wood: The Aftermath")
 					holy_bible.deity_name = pick("Woody", "Andy", "Cherry Flavored Lube")
 				if("Sweet Bro and Hella Jeff: Expanded Edition")
 					holy_bible.deity_name = pick("Sweet Bro", "Hella Jeff", "Stairs", "AH")
 				if("F.A.T.A.L. Rulebook")
 					holy_bible.deity_name = "Twenty Ten-Sided Dice"
+				if("Toilet Humor")
+					holy_bible.deity_name = pick("Skibidi Toilet", "Skibidi Biden")
```

Comment:
I'm not sure if we should have references to real political figures hardcoded.

Reply:
depends on how maintainers feel imo, not much of a loss if it's unwanted though.

Reply:
if only it could check for the presence of Skibidi _at all_ instead of just a specific Skibidi Character

Reply:
I've replaced these with some more generic names, since hardcoding a political figure is probably hell for multiple reasons here